core/src/main/scala/org/apache/spark/h2o/converters/ConverterUtils.scala

```diff
@@ -110,6 +119,13 @@ object ConverterUtils {
     converterContext
   }
 
+  def getReadConverterContext(keyName: String,
```

Comment:
Can you keep only original `getReadConverterContext` function ?
This function will probably get removed anyway after merging of https://github.com/h2oai/sparkling-water/pull/49. Can you please keep the original version and variables like `isExternal` ? Or I can put it back when doing my merge, but seems like extra work :)